.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,40 @@
-## Description
-
-Describe your changes in detail.
-
-## Motivation and Context
-
-Why is this change required? What problem does it solve?
-If it fixes an open issue, please link to the issue here.
-You can use the syntax `close #15213` if this solves the issue #15213
-
-- [ ] I have raised an issue to propose this change ([required](https://github.com/camel-ai/camel/issues) for new features and bug fixes)
-
-## Types of changes
-
-What types of changes does your code introduce? Put an `x` in all the boxes that apply:
+💡 **User Story:**
+As a developer,
+I want to [describe the action or functionality you want to add]
+So that [explain the benefit or value this new functionality will bring]
 
+🛒 **Types of Changes:**
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds core functionality)
```

Comment:
Let's add the following after New feature:
Improvement (non-breaking change which improves existing core functionality)

Reply:
Hi Dmitrii, thank you for your suggestion and sorry for the late reply, have added it